packages/flutter/lib/src/semantics/semantics_event.dart

```diff
@@ -133,3 +133,21 @@ class TapSemanticEvent extends SemanticsEvent {
   @override
   Map<String, dynamic> getDataMap() => const <String, dynamic>{};
 }
+
+/// An event which triggers an announcement of a live region.
+///
+/// This requires that the semantics node has already been marked as a live
+/// region. Only Android (TalkBack) will make a verbal announcement, as long as
```

Comment:
What does iOS do?

Reply:
Nothing :(

Reply:
I would probably spell that out in the doc comment.

Reply:
Never mind. I see it's there now. Was looking at an old diff :(